beachball.config.js

```diff
@@ -1,2 +1,16 @@
-require("./build/releasing/ts-node-register");
-module.exports = require("./build/releasing/index");
+module.exports = {
+    disallowedChangeTypes: ["major"],
+    ignorePatterns: [
+        ".ignore",
+        ".github/",
+        ".prettierrc",
+        ".vscode/",
+        "docs/**",
+        "jest..js",
+        "src/e2e/",
+        "src/tests/",
+        "src/fixtures/**",
+        // This one is especially important (otherwise dependabot would be blocked by change file requirements)
+        "yarn.lock",
```

Comment:
We previously included a configuration to ensure this only affected those things under packages. This will be the most specific config - It seems we are missing parity with the current config. Is that accurate? If a change is made in “sites”, does this expect a change file?

Reply:
@Ibrahimmaga can you help me understand how or why the above has been resolved? There's not a comment or a reference to anything, so from what I can tell - this is still a valid and outstanding question.

Reply:
Tested with the groups change by updating something within sites and didn't hit any issues. I have raised one other comment above that I think is likely something to address here.

Reply:
@Ibrahimmaga I just pulled this down with the groups addition and it worked - is there a reason it's disappeared now?